pkg/operator/mon/mon.go

```diff
@@ -328,18 +329,22 @@ func getNodeInfoFromNode(n v1.Node) (*NodeInfo, error) {
 			break
 		}
 	}
+	nr.Hostname = n.Labels["kubernetes.io/hostname"]
```

Comment:
there is a const value you can reference for this instead of using a hard coded string: `apis.LabelHostname`